tinyagentos/chat/reactions.py

```
@@ -0,0 +1,85 @@
+        # The trace_id on an agent reply points at the user message that triggered it.
+        msg_store = getattr(state, "chat_messages", None)
+        original_text = ""
+        original_id = message.get("id")
+        trace_id = (message.get("metadata") or {}).get("trace_id") or message.get("id")
+        if msg_store is not None and trace_id:
+            try:
+                original = await msg_store.get_message(trace_id)
+                if original:
+                    original_text = original.get("content") or ""
+                    original_id = original.get("id") or original_id
+            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)